lib/http/HttpServer.ts

```diff
@@ -83,14 +83,14 @@ class HttpServer {
   /**
    * Starts the server and begins listening on the provided port.
    */
-  public listen = async (port: number) => {
+  public listen = async (host: string, port: number) => {
```

Comment:
Minor nit but elsewhere in the code for `listen` methods the first parameter is port and second is host, similar to the `server.listen` method that's called below, so I think it'd be better to match that ordering here.